app/components/org-group-form.js

```diff
@@ -0,0 +1,45 @@
+import Component from '@glimmer/component';
+
+import FormMixin from 'open-event-frontend/mixins/form';
+//OrgGroupForm
+export default Component.extend(FormMixin, {
+
+    getValidationRules() {
+        return {
+          inline : true,
+          delay  : false,
+          on     : 'blur',
+    
+          fields: {
+            name: {
+              identifier : 'name',
+              rules      : [
+                {
+                  type   : 'empty',
+                  prompt : this.l10n.t('Please enter your Organizer or Group name')
```

Comment:
Codacy found an issue: [Expected indentation of 14 spaces but found 18.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6000753)